mypy/typeops.py

```diff
@@ -715,8 +715,8 @@ class Status(Enum):
             for name, symbol in typ.type.names.items():
                 if not isinstance(symbol.node, Var):
                     continue
-                # Skip "_order_" and "__order__", since Enum will remove it
-                if name in ("_order_", "__order__"):
+                # Skip "_ignore", "_order_" and "__order__", since Enum will remove it
+                if name in ("_ignore", "_order_", "__order__"):
```

Comment:
Should this have `_ignore_` and not `_ignore`?

Reply:
Fixed.